vpn/client.go

```diff
@@ -23,6 +25,14 @@ var (
 
 	// ErrNotReady is returned when a Read/Write attempt is made before the tunnel is ready.
 	ErrNotReady = errors.New("tunnel not ready")
+
+	// ErrBadProxy is returned when attempting to use an unregistered proxy.
+	ErrBadProxy = errors.New("unknown proxy")
+)
+
+const (
+	// dialTimeoutInSeconds tells how long to wait on Dial
+	dialTimeoutInSeconds = 10
```

Comment:
Maybe this is a little too low; I'd use ~15 seconds here

Reply:
sounds good